src/main/java/seedu/address/model/university/UniqueUniversityList.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.model.university;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.localcourse.LocalCourse;
+import seedu.address.model.localcourse.UniqueLocalCourseList;
+import seedu.address.model.localcourse.exceptions.DuplicateLocalCourseException;
+import seedu.address.model.localcourse.exceptions.LocalCourseNotFoundException;
+import seedu.address.model.university.exceptions.UniversityNotFoundException;
+
+public class UniqueUniversityList implements Iterable<University> {
+    private final ObservableList<University> internalList = FXCollections.observableArrayList();
+    private final ObservableList<University> internalUnmodifiableList =
+            FXCollections.unmodifiableObservableList(internalList);
+
+    public boolean contains(University toCheck) {
+        requireNonNull(toCheck);
+        return internalList.stream().anyMatch(toCheck::isSameUniversity);
+    }
+
+//    public void add(University toAdd) {
```

Comment:
Can I ask why is this commented out? I think if it is not needed, we can remove it

Reply:
i think we are not having the command for adding of universities

Reply:
Can we remove it?

Reply:
yep ive removed it already, pushed the new changes